app/src/main/java/com/yeonae/chamelezone/view/mypage/mycourse/presenter/MyCoursePresenter.kt

```diff
@@ -17,7 +17,7 @@ class MyCoursePresenter(
         repository.getMyCourseList(memberNumber, object : CourseCallBack<List<CourseResponse>> {
             override fun onSuccess(response: List<CourseResponse>) {
                 val items = mutableListOf<MyCourseItem>()
-                for(i in response.indices){
+                for (i in response.indices) {
                     items.add(response[i].toMyCourseItem(response[i]))
```

Comment:
```suggestion
                    items.add(response[i].toMyCourseItem())
```
인자로 넣을 필요가 없는 패턴입니다. 해당 함수 (toMyCourseItem) 를 수정해주세요

Reply:
수정했습니다
a7d77bf